.eslintrc.json

```diff
@@ -0,0 +1,112 @@
+{
+  "env": {
+    "node": true,
+    "es6": true
```

Comment:
Does this "es6" add any value when we have the ecma version down in the parser options?